hotel_housekeeping/report/housekeeping_report.py

```diff
@@ -0,0 +1,63 @@
+# See LICENSE file for full copyright and licensing details.
+
+import time
+from datetime import datetime
+from dateutil.relativedelta import relativedelta
+from odoo import api, fields, models
+from odoo.tools import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class ActivityReport(models.AbstractModel):
+    _name = 'report.hotel_housekeeping.housekeeping_report'
+
+    def get_room_activity_detail(self, date_start, date_end, room_no):
+        activity_detail = []
+        act_val = {}
+        house_keep_act_obj = self.env['hotel.housekeeping.activities']
+
+        act_domain = [('clean_start_time', '>=', date_start),
+                      ('clean_end_time', '<=', date_end),
+                      ('a_list', '=', room_no)]
+        activity_line_ids = house_keep_act_obj.search(act_domain)
+
+        for activity in activity_line_ids:
+            ss_date = datetime.strptime(activity.clean_start_time,
+                                        DEFAULT_SERVER_DATETIME_FORMAT)
+            ee_date = datetime.strptime(activity.clean_end_time,
+                                        DEFAULT_SERVER_DATETIME_FORMAT)
```

Comment:
both could be replaced w/ `fields.Datetime.now()`, then `datetime` and `DEFAULT_SERVER_DATETIME_FORMAT` imports could go away :slightly_smiling_face: